pkg/controller/cronjob/utils_test.go

```diff
@@ -142,6 +144,13 @@ func TestGetParentUIDFromJob(t *testing.T) {
 	{
 		// Case 2: Has UID annotation
 		j.ObjectMeta.Annotations = map[string]string{v1.CreatedByAnnotation: `{"kind":"SerializedReference","apiVersion":"v1","reference":{"kind":"CronJob","namespace":"default","name":"pi","uid":"5ef034e0-1890-11e6-8935-42010af0003e","apiVersion":"extensions","resourceVersion":"427339"}}`}
```

Comment:
Since we plan to remove the annotation in a later release, we should remove this line to make sure things work without it.